ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/acl.md

```diff
@@ -140,6 +140,8 @@ ACL has counting mode `counters per-entry` enabled!
 | 20 | permit ip 10.0.0.0/8 any |
 | 30 | permit ip 192.0.2.0/24 any |
 
+ACL has permit response traffic rule `permit response traffic nat` enabled!
```

Comment:
Please add it in the table like `| - |  permit response traffic nat |` instead. Thanks.

Reply:
Done!